src/ray/common/ray_syncer/ray_syncer.cc

```diff
@@ -355,9 +355,8 @@ ServerBidiReactor *RaySyncerService::StartSync(grpc::CallbackServerContext *cont
         syncer_.sync_reactors_.erase(node_id);
         syncer_.node_state_->RemoveNode(node_id);
       });
-  RAY_LOG(DEBUG) << "Get connection from "
-                 << NodeID::FromBinary(reactor->GetRemoteNodeID()) << " to "
-                 << NodeID::FromBinary(syncer_.GetLocalNodeID());
+  RAY_LOG(INFO) << "Get connection from "
+                << NodeID::FromBinary(reactor->GetRemoteNodeID());
```

Comment:
this is not using WithField, and lost the "to" node id?

Reply:
to is local node id so it doesn't tell us anything. you get it somewhere in the file.